### ISSUE TITLE
Feature Proposal: Show week number

### ISSUE THREAD

**Issue by albinochino** [MantisMountainMobile/TClock-Win10#8]
One of the most used functions of other TClock branches is the show week number option (usually added by `Wi`). It would be great if we could add that, let me know if I can help in any way.

**Comment by themen71** [comment]
I agree the weeknumber and daynumber within a week would be a great and well appreciated addition! Thanks!

**Comment by Screemer1** [comment]
Bumping this, would really be a great feature, just like the original t-clock used to have. :)

**Comment by Dxhs** [comment]
Any development on this? I used to read week-numbers in US and ISO in another T-clock version working on Windows 10 every day at work until win11 hit hard...
